src/components/CustomHeader/CustomHeader.tsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react';
+import CustomButton from '../CustomButton/CustomButton';
+import { useNavigate } from 'react-router-dom';
+import { searchSlice } from '../../stores/reducers/SearchSlice';
```

Comment:
It would be much better to export all actions in the appropriate slice and import it where you need.

Reply:
fixed